### PR TITLE
Use new buf_asm interface for mycpu example plugin

### PR DESCRIPTION
**Pull request by Ace17**

### CODE REVIEW

File: microblaze/asm/asm_microblaze_gnu.c [hunk 1]
```diff
@@ -80,7 +80,7 @@ static int disassemble(struct r_asm_t *a, struct r_asm_op_t *op, const ut8 *buf,
 	disasm_obj.fprintf_func = &buf_fprintf;
 	disasm_obj.stream = stdout;
 
-	op->buf_asm[0] = '\0';
+  r_strbuf_set(&op->buf_asm, "");
```
Comment (radare) [inline microblaze/asm/asm_microblaze_gnu.c]: Wrong indent and add space before (

Comment (Ace17) [reply microblaze/asm/asm_microblaze_gnu.c]: Thanks, fixed.

**Comment by Ace17** [comment]
Wait, there are more space issues. Fixing ....

**Comment by Ace17** [comment]
OK this should be good to go!

**Comment by radare** [comment]
thanks!

**Pull request merged**